package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-filterbar",
-  "version": "2.0",
+  "version": "2.1",
```

Comment:
As mentioned above, major upgrade perhaps?

Reply:
@msxavi I'm happy with any version actually and if you think 3.0 better reflects current changes and improvements we can change it 👌